lib/Conch/Validation.pm

```diff
@@ -14,17 +14,6 @@ Conch::Validation - base class for writing Conch Validations
 	has category    => 'CPU';
 	has description => q/Description of the validation/;
 
-	# Optional schema to validate $input_data before `validate` is run.
```

Comment:
This was a useful feature, I think?  Although perhaps not done in the best way.  I wonder if we could still just give the user access to the json validator in this class, and (optionally) allow them to specify their own schema name to do validations against, as that can get a lot of checks done faster/more concisely than expressing the same thing in perl, before moving on to more sophisticated checks that can't be expressed nicely in the schema.

(I'm not proposing making a change now, but instead considering it for future enhancements.)

Reply:
Let's see how the rework comes out first. I suspect validations are going to look very different in the near future.